README.md

```diff
@@ -37,6 +37,7 @@ The Chocolatey Design System has all information relating to Astro Components an
 * Learn about Astro and how to use [Components in `.mdx` and `.astro`](https://design.chocolatey.org/foundations/astro) file types.
 * Learn how to use the [`<Callout />` Component](https://design.chocolatey.org/components/callouts) to display notes and important information.
 * Learn how to use the [`<CollapseButton />` Component](https://design.chocolatey.org/collapse-button) to display a button that triggers a collapsed element.
+* Learn how to use the [`<CodeBlock />` and `<InputForCodeBlock />` Components](https://design.chocolatey.org/code-block) to display a code block with variables that can be replaced with values from inputs.
```

Comment:
```suggestion
* Learn how to use the [`<DynamicCodeBlock />` and `<DynamicCodeBlockInput />` Components](https://design.chocolatey.org/components/dynamic-code-block) to display a code block with variables that can be replaced with values from inputs.
```

I think we should include `Dynamic` in the component name here as well, that way when it's imported it's clear it a dynamic one. There might be some time in the future that we just add a general `CodeBlock` component so we don't want to get them confused.